### PR TITLE
added yaml to allow for setting certs from other namespace

### PR DESCRIPTION
**Pull request by scoyle391**
This adds info in the "Running Ambassador" doc page.

### CODE REVIEW

File: reference/running.md [hunk 1]
```diff
@@ -98,7 +98,21 @@ env:
...
-With the Ambassador Edge Stack, if you set `AMBASSADOR_NAMESPACE` or `AMBASSADOR_SINGLE_NAMESPACE`, set it in deployment container. 
+With the Ambassador Edge Stack, if you set `AMBASSADOR_NAMESPACE` or `AMBASSADOR_SINGLE_NAMESPACE`, set it in deployment container.
+
+If you want to set a certificate for your `TLScontext` from another namespace, you can use the following:
+
+```yaml
+env:
+- name: AMBASSADOR_SINGLE_NAMESPACE
+  value: "YES"
+- name: AMBASSADOR_CERTS_SINGLE_NAMESPACE
```
Comment (alexgervais) [inline reference/running.md]: No `value:` property? It might work... I'm just not used to seeing this!

Comment (scoyle391) [reply reference/running.md]: That's the info the customer put in. @kflynn understands this a bit more than me however, so maybe he can chime in?

Comment (kflynn) [reply reference/running.md]: I'd put `value: "YES"` like `AMBASSADOR_SINGLE_NAMESPACE` has.

Comment (scoyle391) [reply reference/running.md]: Okay, added value in. Please review again @kflynn

**Comment by kflynn** [comment]
@scoyle391 Can you resolve conflicts and ping me again? Sorry.

**Pull request merged**